.buildkite/pipeline.yml

```
@@ -92,3 +92,29 @@ steps:
         notify:
           - github_commit_status:
               context: "install_swiftpm_dependencies Tests: Xcode Workspace and Project - Explicit"
+
+  - group: ":github: pr_changed_files Tests"
```

What's a bit funny and meta is that once this new `pr_changed_files` helper lands in `trunk`, we could consider use it on itself to only run its unit tests if the script itself has changed, i.e.

```suggestion
  - group: ":github: pr_changed_files Tests"
    if: build.env('RUN_PR_CHANGED_FILES_TESTS')
```
After adding `RUN_PR_CHANGED_FILES_TESTS=$(pr_changed_files --includes "bin/pr_changed_files" "tests/pr_changed_files/*")` to the `shared-pipeline-vars` of this repo 😛

---

YES! I was thinking exactly the same 😄